NEWS.md

```diff
@@ -48,6 +48,7 @@
   `default_linters` (#693, #695, @AshesITR).
 * `lint_package()` now lints files in the `demo` directory by default (#703, @dmurdoch).
 * `commented_code_linter()` uses the parse tree to find comments, eliminating some false positives (#451, @AshesITR)
+* `trailing_blank_lines_linter()` now also lints files without a terminal newline (#675, @AshesITR)
```

Comment:
corrected this I think -- original wording sort of implies the linter always triggers (when there is terminal whitespace & when there is not)